test/playwright.config.ts

```diff
@@ -10,10 +10,10 @@ import path from "path"
 //   yarn test:e2e --grep login       # Run tests matching "login"
 const config: PlaywrightTestConfig = {
   testDir: path.join(__dirname, "e2e"), // Search for tests in this directory.
-  timeout: 60000, // Each test is given 60 seconds.
+  timeout: 30000, // Each test is given 25 seconds.
```

Comment:
```suggestion
  timeout: 30000, // Each test is given 30 seconds.
```

Although we might want to remove the comment since otherwise we have to update it every time we change the timeout and doing the math is pretty easy.  We could do `// In milliseconds.` in case the unit is not obvious.

Reply:
Good idea, I'm guessing I copied that from Playwright. I'll update.